README.md

```diff
@@ -20,23 +20,27 @@ Julia interface to [SCIP](http://scip.zib.de) solver.
 
 ## Installation
 
-The SCIP.jl package requires SCIP to be installed.
+The SCIP.jl package requires [SCIP](http://scip.zib.de/) to be installed.
 
 [Download](http://scip.zib.de/download.php?fname=scipoptsuite-3.2.1.tgz) the
-SCIP Optimization Suite, copy the
-[patched file](http://scip.zib.de/download/bugfixes/scip-3.2.1/Makefile.doit)
```

Comment:
The best (most up-to-date) installation instructions are those that are used in the [Travis build](https://github.com/SCIP-Interfaces/SCIP.jl/blob/master/.travis.yml) :wink: